src/router/SplashRouter.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react'
+
+import SplashPage from '@/pages/signup/splash'
+
+import { Outlet } from 'react-router-dom'
+
+const SplashRouter = () => {
+  const isReturningUser = sessionStorage.getItem('isReturningUser')
+  const [isSplashOn, setIsSplashOn] = useState(true)
+
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      setIsSplashOn(false)
+      sessionStorage.setItem('isReturningUser', 'true')
+    }, 1000)
+    return () => clearTimeout(timer)
+  }, [])
+
+  if (!isReturningUser) {
+    if (isSplashOn) {
+      return <SplashPage />
+    } else {
+      return <Outlet />
+    }
+  } else {
+    return <Outlet />
+  }
+}
```

Comment:
P3: 조건문을 좀 더 단순하게 처리할 수 있을 것 같습니다!
```suggestion
  if (isSplashOn && !isReturningUser) {
    return <SplashPage />;
  } else {
    return <Outlet />;
  }
}
```